css/style.css

```diff
@@ -0,0 +1,30 @@
+nav {
+  text-align: center;
+  background-color: yellow;
+
+}
+
+h1 {
+  font-family: sans-serif;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+p {
```

Comment:
@dbklein6 just a heads up, you could have combined the `<p>` and `<a>` tag styling for the font like this:

```css
p,a {
  font-family: georgia;
}
```